.devcontainer/devcontainer.json

```diff
@@ -2,7 +2,7 @@
     "name": "AVD Development",
     "build": {
         "dockerfile": "Dockerfile",
-        "args": { "VARIANT": "3.8-v2.0" },
```

Comment:
Unrelated change

Reply:
Agreed - this was related to my local environment. Reverting